encoder.go

```diff
@@ -193,10 +193,17 @@ func (e *Encoder) buildHeader(m Metric) error {
 
 		// Some keys and values are not encodeable as line protocol, such as
 		// those with a trailing '\' or empty strings.
-		if key == "" || value == "" {
+		if value == "" {
 			continue
 		}
 
+		switch key {
+		case "":
+			continue
+		case "_measurement", "_field", "time":
```

Comment:
Should this be "_time" not "time"?

Reply:
I confirmed with @stuartcarnie that it is `time` 👍

Reply:
`time` is correct and is already rejected in storage:

https://github.com/influxdata/influxdb/blob/8f6701d4b1a12c7a5dd2f0adc92ea5ab4967e5f8/storage/engine.go#L487-L491